src/PowerShellEditorServices.Hosting/EditorServicesLoader.cs

```diff
@@ -406,5 +409,14 @@ private void ValidateConfiguration()
                 throw new ArgumentNullException(nameof(_hostConfig.PSHost));
             }
         }
+
+        private static object GetPSVersion()
+        {
+            using (var pwsh = SMA.PowerShell.Create())
```

Comment:
Curious if this is the same as `[runspace]::DefaultRunspace`... probably, right? Since you're getting the LanguageMode.